packages/ifpack2/test/unit_tests/Ifpack2_UnitTestOverlappingRowMatrix.cpp

```diff
@@ -109,6 +110,145 @@ typedef Tpetra::global_size_t GST;
 } while (false)
 
 
+
+
+/***********************************************************************************/
+template<class MatrixClass, class MultiVectorClass>
```

Comment:
It's not your fault, but it's annoying that `KokkosSparse::CrsMatrix` doesn't work like `Kokkos::View`, with a parameter pack `...`.  That would let us write `KokkosSparse::CrsMatrix<T, ...>` instead of this excessively general `MatrixClass`.  On the other hand, we could probably be more specific for `MultiVectorClass` here.

Reply:
Because gitlab doesn't have a pitchfork emoji  ⛏     :P